testing/Makefile

```diff
@@ -1,18 +1,17 @@
-
-DIAG=diag.log
 BTEST=../../btest/btest
+DIAG=diag.log
 
 all: cleanup btest-verbose
 
 # Showing all tests.
 btest-verbose:
-	@$(BTEST) -j -f $(DIAG)
+	@test -f $(BTEST) && bt=$(BTEST) || bt=btest; $$bt -j -f $(DIAG)
```

Comment:
Would `-x` be better than `-f` here? That tests that it not only exists, but is also executable.

Reply:
Also, out of curiosity, why the double `$$` in `$$bt`?

Reply:
[Escaping for `$`](https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html#index-rules_002c-and-_0024) so `bt` is evaluate in the shell session 🦄 .

Reply:
Good point re `-x`, will do Tim. Benjamin's on it re `$$`. I don't think this is super elegent btw — I initially tried to express this properly in the Makefile but it's a pain to make portable, so settled for this.